tests/Unit/CountryValidatorTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Unit;
+
+use App\Http\Requests\CountryRequest;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Validator;
+use Tests\TestCase;
+
+class CountryValidatorTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testCountryRequestValidation(): void
+    {
+        $data = [
+            "name" => "Croatia",
+            "lat" => 83.752131,
+            "lon" => "-177.71159",
+            "iso" => "cr",
+        ];
+
+        $validator = Validator::make($data, (new CountryRequest())->rules());
+
+        $this->assertTrue($validator->fails());
```

Comment:
It will be better to use `$this->expectException()` ad the begining of this method than `assertTrue`.

Reply:
$validator->fails() doesn't throw an exception, it returns a boolean value: validation fails or not.

Reply:
Validation can be tested in feature test, by just providing invalid data. So this Unit is not needed in my opinion.